src/components/MainContent/MainContent.jsx

```diff
@@ -0,0 +1,20 @@
+import logo404 from '../../images/logo404.png'; 
+import './MainContent.css';                 
+import React from 'react';
+
+class MainContent extends React.Component {
```

Comment:
you can create separate component for content of NotFoundPage, but in this case it's better to specify the name (smth like NotFoundPageContent)